biz.aQute.bndlib/src/aQute/bnd/plugin/jpms/JPMSModuleInfoPlugin.java

```diff
@@ -85,29 +86,39 @@
 		SYNTHETIC(ACC_SYNTHETIC),
 		MANDATED(ACC_MANDATED);
 
-		public static Access parse(String input) {
+		public static Set<Access> parse(String input) {
 			switch (input) {
 				case "OPEN" :
 				case "open" :
```

Comment:
Would it be better to switch on `input.toLowerCase()` and then you can get rid of the duplicate case statements as well as make the parse method that little bit more forgiving? Or are there strict case requirements for the input string?